source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -152,21 +366,138 @@ provision the following networking API entities, represented in yaml:
+  os_network:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: netv6
+  register: ipv6_network
+
+- name: attach a subnet to the logical network
+  os_subnet:
+    cloud: "{{ cloud_name }}"
+    state: present
+    network_name: netv6
+    name: subnet_ipv6
+    ip_version: 6
+    cidr: bef0:1234:a890:5678::/64
+    gateway_ip: bef0:1234:a890:5678::1
+    ipv6_address_mode: dhcpv6_stateless
+  register: dhcpv6_subnet
+
+- name: create a router to send RA messages
+  os_router:
+    cloud: "{{ cloud_name }}"
+    state: present
+    name: router0
+    interfaces:
+      - "{{ dhcpv6_subnet.id }}"
+~~~~~
+
+## UI considerations
+
+The subnet ipv6_address_mode parameter should be added to the UI, since
+updates to existing subnets are not possible through the UI, and updates to the
+ipv6_address_mode parameter are not allowed, as per the
+[networking API](https://developer.openstack.org/api-ref/network/v2/?expanded=create-subnet-detail,update-subnet-detail#update-subnet). Until the UI is updated,
+the subnet should be created via ovirt-provider-ovn RESTful API or ansible.
+
+It is outside the scope of this feature to implement the UI changes that allow
+logical routers to be provisioned, and the REST API - or its ansible wrapper
+should be used instead. An ansible example is available
+[here](#configuring-stateless-dhcpv6-on-the-subnet).
+
 ## Testing
 
-Two scenarios are recommended:
-  - single IPv6 subnet
-  - multiple IPv6 subnets connected by router
+Two scenarios are recommended, each with two different configurations, leading
+to four different tests:
+  - single IPv6 subnet using dhcpv6-stateful configuration
+  - single IPv6 subnet using dhcpv6-stateless configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateful configuration
+  - multiple IPv6 subnets connected by router using dhcpv6-stateless configuration
+
+All four scenarios on EL8 guest VMs require a router, whereas on EL7 guest VMs
+only the dhcpv6-stateless configurations require it.
+
+### Configuration mechanism used for each option
+
+|        Option Name        | dhcpv6-stateful | dhcpv6-stateless |
+|:-------------------------:|:---------------:|:----------------:|
+| DNS recursive name server |       DHCP      |       DHCP       |
+|     Domain Search List    |       DHCP      |       DHCP       |
+|       Network Prefix      |       None      |        RA        |
+|         IP address        |       DHCP      |  Auto generated  |
+|   Network Prefix Length   |        RA       |        RA        |
+|            MTU            |        RA       |        RA        |
+|      Default Gateway      |        RA       |        RA        |
+
+### Supported use cases per dhcpv6 configuration
+
+| Tables                  |           dhcpv6-stateful           | dhcpv6-stateless |
+|-------------------------|:-----------------------------------:|:----------------:|
+| fixed IPs               |              supported              |   not supported  |
```

Comment:
you could insert a line for dynamic IPs

Reply:
I could, but I think it's heavily implied that dynamic addresses work on all of them, in different ways. 

That has been thoroughly covered in the document so far.

Reply:
Do dynamic addresses work for dhcpv6-stateful?

Reply:
they do, *if* the network prefix length is 64.

That's explained in https://github.com/oVirt/ovirt-site/blob/bc62195cf28643ed2cb696eac46537a4c6804a9b/source/develop/release-management/features/network/ipv6_subnets.html.md#dhcpv6 .

I plan to add a comment to [0] indicating that for prefix lengths = 64 dynamic addresses are possible.

[0] - https://github.com/oVirt/ovirt-site/blob/bc62195cf28643ed2cb696eac46537a4c6804a9b/source/develop/release-management/features/network/ipv6_subnets.html.md#allowed-prefix-lengths

Reply:
Done